index.js

```diff
@@ -78,7 +78,7 @@ function pipeLogsToTerminal() {
   });
 
   afterEach(function() {
-    if (this.currentTest.state !== 'passed') {
+    if (this.currentTest.state !== 'passed' || config && config.forcePrintLogs) {
```

Comment:
I thing a better name would be `printLogs` that could have 'on-failure' or 'always' and by default to be 'on-failure'. In this way it's more descriptive. Add the readme and it's good to go, tomorrow I'll add tests for this as well and we have a nice release.

Reply:
All done.